server/api/events/user_message_events.js

```diff
@@ -0,0 +1,68 @@
+var express = require('express');
+var router = express.Router();
+var Moment = require('moment-timezone');
+const ApiValidator = require("./../api_validator");
+const UserMessageEvents = require("../../models/index").UserMessageEvents;
+const Message = require("../../models/index").Message;
+
+const VALIDATOR = new ApiValidator({
+  state: {

```

Comment:
User message event does not have a state field.  Going back to the spec, the fields are:
messageId  int, required
clientTime, string, required
type, string, default value is "open"